ddsketch/ddsketch.py

```diff
@@ -59,92 +69,96 @@ def name(self):
 
     @property
     def num_values(self):
-        return self._count
+        return self.count
 
     @property
     def avg(self):
-        return float(self._sum) / self._count
+        return self._sum / self.count
 
     @property
     def sum(self):
         return self._sum
 
-    def get_key(self, val):
-        if val < -self.min_value:
-            return -int(math.ceil(math.log(-val) / self.gamma_ln)) - self.offset
-        elif val > self.min_value:
-            return int(math.ceil(math.log(val) / self.gamma_ln)) + self.offset
-        else:
-            return 0
-
     def add(self, val):
         """Add a value to the sketch."""
-        key = self.get_key(val)
-        self.store.add(key)
+        if val > self.min_value:
```

Comment:
Why do we still need `min_value` instead of comparing `val > 0`? Might be worth a comment

Reply:
👍 will do. the min_value is basically the smallest possible float right now.